### PR TITLE
Add post-render step to push skaffold updated manifests to app-config-repo.

### PR DESCRIPTION
**Pull request by yuwenma**
…onfig-repo.

### CODE REVIEW

File: starter-repos/app-template/skaffold/render.yaml [hunk 1]
```diff
@@ -36,6 +46,15 @@ skaffold-render:
...
-    paths:
-      - config/envs
+
+    - |
+      # If files have changed, commit them back to the env repo in the staging branch
+      if [ -z "$(git status --porcelain)" ]; then
+        echo "No changes found in env repository."
+      else
+        git add config
+        git commit -m "update built manfiests"
```
Comment (Liujingfang1) [inline starter-repos/app-template/skaffold/render.yaml:56]: In the commit message, can you add the commit sha that triggers the current job?

**Review by nan-yu — APPROVED** [review]
/lgtm

File: starter-repos/app-template/skaffold/render.yaml (hunk 1 above)
Comment (yuwenma) [reply starter-repos/app-template/skaffold/render.yaml]: Done. it looks like this. https://screenshot.googleplex.com/6MQSeaK7v5TCnTZ.png

**Review by janetkuo — APPROVED** [review]

**Pull request merged**